framework/wazuh/core/cluster/master.py

```diff
@@ -482,11 +482,7 @@ def process_sync_error_from_worker(self, command: bytes, error_msg: bytes) -> Tu
         bytes
             Response message.
```

Comment:
Improve the result information

Reply:
I think that for consistency with the other docstrings (it is repeated in many places), it will be better not to change it in this issue. It would be better to open a new one to improve the docstrings if necessary.

Reply:
Agreed